debian/changelog.in

```diff
@@ -2,7 +2,7 @@
 
   * New upstream version.
 
- -- BalaBit Development Team <devel@balabit.hu>  @CURRDATE@
+ -- BalaBit Development Team <syslog-ng-ose@balabit.com>  @CURRDATE@
 
```

Comment:
feedback@syslog-ng.org as an email address?

or an alternate address with the syslog-ng.org domain?